### ISSUE TITLE
Centrum zniżek z poziomu API

### ISSUE THREAD

**Issue by mateuszcom** [allegro/allegro-api#9421]
Dzień dobry, czy z poziomu API istnieje możliwość dostępu do danych zawartych w Centrum Zniżek? https://allegro.pl/moje-allegro/sprzedaz/centrum-znizek

**Comment by Lukasz-Zurek** [comment]
@mateuszcom Nie, nie udostępniamy takich funkcjonalności w API.

**Comment by DanielCyran** [comment]
Dzień dobry,
również potrzebujemy dostępu w API do informacji o aktywnych zniżkach i pakietach przypisanych do konta sprzedawcy (np. pakiet powitalny).

Jest to potrzebne do automatycznej reakcji systemu na zakończenie lub zmianę pakietu, np. przeliczenia prowizji, korekty cen ofert lub aktualizacji kosztów w systemach zewnętrznych.
Bez tych danych automatyzacja rozliczeń jest nieprecyzyjna i wymaga ręcznej kontroli.

Czy planowane jest udostępnienie read-only endpointu lub eventu/webhooka informującego o zmianach w pakietach promocyjnych konta?

**Comment by Lukasz-Zurek** [comment]
@DanielCyran Widzę, że właśnie utworzyłeś nowy wątek/sugestię na [Github Discussions](https://github.com/allegro/allegro-api/discussions/12949) i bardzo Ci dziękujemy za ten pomysł. Opisana przez Ciebie potrzeba jest uzasadniona. Nie mamy co prawda planów związanych z tymi funkcjonalnościami, jednak ta przestrzeń jest właśnie po to, by to zmienić.

**Comment by DanielCyran** [comment]
Tak, utworzyłem nową dyskusję celowo, ponieważ jest to istotna luka funkcjonalna w REST API.

Potrzeba dostępu do danych z Centrum Zniżek była już zgłaszana w 2024 roku i od tego czasu nie pojawiło się żadne rozwiązanie ani alternatywa po stronie API. Tymczasem brak tych informacji uniemożliwia poprawną automatyzację rozliczeń oraz reakcję systemów integrujących się z Allegro na zmianę warunków kosztowych konta sprzedawcy.

W praktyce oznacza to, że integracje nie są w stanie:
- wiarygodnie określić momentu zakończenia pakietów promocyjnych (np. pakietu powitalnego),
- poprawnie przeliczać prowizji i kosztów sprzedaży,
- utrzymać spójności danych kosztowych w systemach zewnętrznych.

Jest to funkcjonalność o charakterze podstawowym z punktu widzenia integratorów i automatyzacji procesów sprzedażowych, dlatego temat został ponownie podniesiony w GitHub Discussions z nadzieją na realne rozważenie jego wdrożenia.
